src/components/partials/user-editor/user-editor.component.js

```diff
@@ -222,6 +229,8 @@ class UserEditor extends Component {
           userGroup: "",
         };
 
+    const isAgencyOrGlobal = ["agency", "global"].includes(authService.userRole);
+
```

Comment:
added a boolean here to use inside the form whether to display the Select or the read-only TextField